README.md

```diff
@@ -42,6 +42,8 @@ bareos_filesets:
   - name: FilesetFoo
     description: "Backup Foo"
     include_file: /home/foo
+    exclude_file:
+      - /home/foo
```

Comment:
Can you make the example consistant so the `include_file` and `exclude_file` don't use the same path?

```suggestion
      - /home/bar
```

Reply:
I understand, that's true !